app/services/contact_details_saver.rb

```diff
@@ -16,9 +16,9 @@ def save(form)
 
   def persist_answers(form)
     @selected_answer_store.store_selected_answers(
-      :contact_details,
-      full_name: form.full_name,
-      telephone_number: form.telephone_number,
+      'contact_details',
+      'full_name' => form.full_name,
+      'telephone_number' => form.telephone_number
```

Comment:
Would this benefit from a comment in the commit message to explain it?

Reply:
Yes, I've added a comment about this. Also added a card around fixing `SelectedAnswerStore` so that stringified/symbolized keys behave the same way.